Userland/cal.cpp

```diff
@@ -0,0 +1,46 @@
+#include <stdio.h>
+#include <time.h>
+
+int dayOfWeek(int day, int month, int year)
+{
+    const int seekTable[] = { 0, 3, 2, 5, 0, 3, 5, 1, 4, 6, 2, 4 };
```

Comment:
If you do this, it'll allocate the array on the stack and then copy over the data from the `.rodata` section (or even from the code section itself). Add `static`, and it would give you a pointer to the data in `.rodata` directly without copying it.